src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -505,6 +506,13 @@
     Collection<StatisticFormat> getStatisticFormats();
 
     /**
+     * Gets a collection of all available {@link Achievement}s.
+     *
+     * @return An immutable collection containing all available achievements
+     */
+    Collection<Achievement> getAchievements();
+
```

Comment:
There hasnt been a getId method on Achievements. And i wasnt sure whether it should be exposed so i skipped that. See description.